CONTRIBUTING.md

```diff
@@ -3,6 +3,13 @@
 We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
+## How to build
+> Only windows and macos platforms are supported at the moment
```

Comment:
"Windows and macOS". Let's make sure the capitalization is consistent, there are a couple other instances.

Reply:
done